src/js/view/view.js

```diff
@@ -828,6 +832,66 @@ function View(_api, _model) {
         displayClickHandler.revertAlternateClickHandlers();
     };
 
+    const _pip = function(model, state) {
+        // Unmute the video so volume can be adjusted with native controls in fullscreen
```

Comment:
What is this comment referring to?

Reply:
Nothing - bad comment that I forgot to remove!